app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
For custom exception use "Exception" instead of "BaseException" as you do it for VaccineError. All usee-definded exceptions shoud be derived from class "Exception"